src/index.js

```diff
@@ -72,55 +74,61 @@ if (window.location.search) {
         HueBridgeList.load();
         const bridge = HueBridge.getById(bridgeId);
         if (bridge && bridge.properties.username) {
-          bridge.properties.remote = true;
+          // If the bridge was authorized before, e.g. locally, add remote properties.
+          bridge.properties = {
+            ...bridge.properties,
+            ...bridgeOAuthProperties,
+            remote: true,
+          };
           bridge.store();
           oauthSuccess(state);
         } else {
-          return fetch(`/bridge/0/config`, {
+          // If the bridge was never seen before, authorize through remote API.
+          fetch(`/bridge/0/config`, {
             method: 'PUT',
             body: JSON.stringify({
               linkbutton: true,
             }),
             headers: {
-              Authorization: `Bearer ${settings.accessToken}`,
+              Authorization: `Bearer ${bridgeOAuthProperties.accessToken}`,
               'content-type': 'application/json',
             },
-          });
+          })
+            .then((response) => {
+              return response.json();
+            })
+            .then((json) => {
+              console.log(json);
```

Comment:
probably should remove console.logs..

Reply:
I use it for debugging production traffic. I need a better way to manage this, like it should only show up when I turn on a flag.